cwms-data-api/src/main/java/cwms/cda/data/dao/LocationLevelsDaoImpl.java

```diff
@@ -214,10 +214,13 @@ public void deleteLocationLevel(String locationLevelName, ZonedDateTime zonedDat
                                     String officeId, Boolean cascadeDelete) {
         try {
             Timestamp date;
+            String timeZone;
             if (zonedDateTime != null) {
-                date = Timestamp.from(zonedDateTime.toLocalDateTime().atZone(zonedDateTime.getZone()).toInstant());
+                date = Timestamp.from(zonedDateTime.toInstant());
+                timeZone = zonedDateTime.getZone().getId();
```

Comment:
This is creeping me out.  Is the getZone() thing right?  I guess I'd need to try it in the debugger.  I would have thought it was always utc.

Reply:
oh, you changed the date calculation too.   I see it now

Reply:
yeah I guess you're right, since we're doing the time zone processing already higher up